pkgs/by-name/ap/apriltag/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  opencv4,
+  python3,
```

```suggestion
  python3Packages,
```
You're accessing attributes only present on the Python package set, so you need `python3Packages`.